### ISSUE TITLE
Connect to Node Fails over Tor v3

### ISSUE THREAD

**Issue by frankhinek** [LN-Zap/zap-desktop#3620]
## Description

I recently built a bitcoind/LND node which I connect to over Tor using a v3 address.  I'm able to connect remotely and manage the node using Zap for iOS, but Zap Desktop on macOS fails to connect.  The "Starting Tor proxy" message and oscillating gray/orange gradient background loops endlessly.

**_NOTE_**: Although the steps below specify connecting to LND's REST Tor address and service port (8080), l've tried connecting to the RPC Tor address and port (10009) as well.  The same behavior is observed.

## Expected Behavior

1. Clone and build `lndconnect` from github.com/LN-Zap/lndconnect
2. Use `lndconnect` to generate the LND connect URL:

    ```bash
    ./lndconnect --lnddir=/var/db/lnd -j --host=myonionaddressis.onion --port=8080
    ```
3. Launch Zap Desktop and paste in the LND connect URL.
4. Tor proxy is started and Zap Desktop connects to the node successfully.

## Actual Behavior

1. Clone and build `lndconnect` from github.com/LN-Zap/lndconnect
2. Use `lndconnect` to generate the LND connect URL:

    ```bash
    ./lndconnect --lnddir=/var/db/lnd -j --host=myonionaddressis.onion --port=8080
    ```
3. Launch Zap Desktop and paste in the LND connect URL.
4. "Starting Tor proxy" is displayed with an oscillating gray/orange background that loops endlessly.

I've let it run for an hour, and it never times out.  No error message is ever displayed.  While this is going on, two (2) `Zap Helper` processes consume about 50% of a CPU on my MacBook Pro. 

## Your Environment

- Zap Desktop version: 0.7.2 beta
- Operating System and version: macOS 10.15.6
- LND version: 0.10.2-beta and 0.11.0-beta (I've tested with both -- fails in the exact same way)

**Comment by mayankchhabra** [comment]
I'm facing the same issue.

Zap on iOS connects flawlessly to lnd's REST API over Tor, but Zap on desktop is forever stuck on the connection screen when trying to access LND's gRPC over Tor:

![image](https://user-images.githubusercontent.com/10330103/93428692-85888900-f8dd-11ea-8278-920d23d1bbc1.png)
_(nevermind the onion url, it's a regtest instance)_

**Comment by mrfelton** [comment]
We don't currently bundle the tor binary with zap - it needs to be installed on your system. Can you confirm that you have tor installed and available on your execution path? Thanks.

**Comment by frankhinek** [comment]
@mrfelton Yes, tor is installed.  However, whether or not it is the “PATH” for Zap Desktop depends on which directories Zap is looking at.

```
❯ which  tor
/usr/local/bin/tor

~
❯ tor --version
Tor version 0.4.3.6.
```

Feedback is that it would be great to provide some kind of warning or notice to the user that they need to install tor on their own.

**Comment by ProofOfKeags** [comment]
We are also experiencing this on Zap 0.7.2 on Linux, only in the Linux case we aren't getting any messaging around whether or not the tor proxy is attempting to be started. We have the Tor daemon running at version 0.4.2.7, so maybe it is worth upgrading but I don't know of any meaningful change between those two versions that could explain why Zap is not messaging that it is attempting to connect to the proxy.

**Comment by kevmodrome** [comment]
Any workaround for this?

**Comment by dmytroleonenko** [comment]
Almost one year and still nothing? Is anybody going to fix it?
